lib/registry.js

```diff
@@ -7,6 +7,7 @@ See LICENSE file.
 var request = require('request');
 var url = require('url');
 var config = require('./config.json');
+var lodash = require('lodash');
```

Comment:
Do we really need all of `lodash` for this? Couldn't we just use [`lodash.clonedeep`](https://www.npmjs.com/package/lodash.clonedeep) instead?

We would also just do a poor mans clone too without the need of another module:

```js
parts.push({
    version: name,
    json: JSON.parse(JSON.stringify(json.versions[tag]))
});
```